pyecasound/ecacontrol.py

```diff
@@ -1,6 +1,4 @@
-"""Native Python ECI (ecasound control interface) implementation
-   Can be used to replace the C implementation 'pyecasound.so'.
-"""
+"""Native Python ECI (ecasound control interface) implementation"""
```

Comment:
@marclava Ack. Can you update the git commit message for this as well. Change looks good.

Reply:
Actually a quick update -- this import is still needed. Without this, ecacontrol.py fails with python2.7 (tested with python 2.7.16).